codalab/bundles/run_bundle.py

```diff
@@ -35,9 +35,9 @@ class RunBundle(DerivedBundle):
 
     METADATA_SPECS.append(MetadataSpec('actions', list, 'Actions (e.g., kill) that were performed on this run.', generated=True))
 
-    METADATA_SPECS.append(MetadataSpec('time', float, 'Amount of time (seconds) used by this run (total).', generated=True, formatting='duration'))
-    METADATA_SPECS.append(MetadataSpec('time_user', float, 'Amount of time (seconds) by user.', generated=True, formatting='duration'))
-    METADATA_SPECS.append(MetadataSpec('time_system', float, 'Amount of time (seconds) by the system.', generated=True, formatting='duration'))
+    METADATA_SPECS.append(MetadataSpec('time', float, 'Amount of time (seconds) used by this run command in total (user+system).', generated=True, formatting='duration'))
```

Comment:
Just to confirm, this doesn't include the CodaLab preparing/finalizing time?

Reply:
Actually, from below, it seems like `time` is `time_total`, which is some sort of wall clock time.  Should probably have a comment somewhere that explains this in more detail.